src/main/java/hudson/remoting/ClassFilter.java

```diff
@@ -86,14 +86,26 @@ public final Class check(Class c) {
      * A set of sensible default filtering rules to apply,
      * unless the context guarantees the trust between two channels.
      */
-    public static final ClassFilter DEFAULT = createDefaultInstance();
+    public static final ClassFilter DEFAULT;
+
+    static {
+        try {
+            DEFAULT = createDefaultInstance();
+        } catch (ClassFilterException ex) {
+            LOGGER.log(Level.SEVERE, "Default class filter cannot be initialized. Remoting will not start", ex);
```

Comment:
I throw the unchecked exception. `ExceptionInInitializerError ` is definitely better though